x/mock/simulation/random_simulate_blocks.go

```diff
@@ -274,6 +284,27 @@ func runQueuedOperations(queueOperations map[int][]Operation, height int, tb tes
 	return 0
 }
 
+// nolint: errcheck
```

Comment:
We do want this passing errcheck though. Is there a specific line/error your trying to skip?